tests/test_models.py

```diff
@@ -1,28 +1,33 @@
 import json
+import logging
 import os
 
 from asnake.client import ASnakeClient
 import pytest
 import requests_mock
+import structlog
 
 from asaps import models
 
 
+logger = structlog.get_logger()
+
+
 @pytest.fixture
 def as_ops():
     client = ASnakeClient(baseurl='mock://example.com', username='test',
                           password='test')
-    as_ops = models.AsOperations(client)
+    as_ops = models.AsOperations(client, logger)
     return as_ops
 
 
-def test_get_record(as_ops):
+def test_get_record(as_ops, logger):
```

Comment:
Once you've removed the logger as an argument to the various functions, you won't need to include the `logger` fixture here, or in any of the tests. The fixture you added in conftest.py will automatically configure the logger for testing.